apps/web/src/components/RightSideBarActivities.tsx

```diff
@@ -21,6 +24,8 @@ export const RightSideBarActivites: React.FC = () => {
       setItems(newCareActivity);
       //Set checked ids to previously selected ids when care activity changes
       values.careActivities = _.filter(values.careActivityBundle[values.careActivityID]);
+      //Empty array when careActivityID changes
+      values.checked = [];
```

Comment:
I think I've updated the name to careActivities. Kindly update and test the flow

Reply:
Updated to current state.